app/views/user_mentee_application_alert_mailer/notify_for_application_submission.html.erb

```diff
@@ -0,0 +1 @@
+<p>Application: <%= link_to "Here", user_mentee_application_url(@user_mentee_application) %>.</p>
```

Comment:
this mail is meant for email alert for admins. I didn't think it needs any more messaging than this. but if we do I'll open another pr with it